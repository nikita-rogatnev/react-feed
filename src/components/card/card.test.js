import React from 'react';
import renderer from 'react-test-renderer';

import Card from './card';

it(`Card renders correctly`, () => {
  const rendered = renderer
    .create(<Card
      key={0}
      name='test-name'
      screen_name='test-screen-name'
      url='test-url'
      description='test-description'
      favourites_count='test-favourites-count'
      profile_image_url_https='test-image'
    />).toJSON();
  expect(rendered).toMatchSnapshot();
});
