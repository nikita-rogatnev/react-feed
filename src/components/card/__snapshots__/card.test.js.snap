// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<article
  className="card"
>
  <header
    className="card__header"
  >
    <h1
      className="card__title"
    >
      <img
        alt="test-screen-name"
        className="card__image"
        src="test-image"
      />
      <a
        className="card__name"
        href="test-url"
        rel="noopener noreferrer"
        target="_blank"
      >
        test-name
      </a>
      <a
        className="card__nickname"
        href="test-url"
        rel="noopener noreferrer"
        target="_blank"
      >
        @
        test-screen-name
      </a>
    </h1>
  </header>
  <div
    className="card__body"
  >
    <p>
      test-description
    </p>
  </div>
  <footer
    className="card__footer"
  >
    <span
      className="card__likes"
    >
      ‚ù§
    </span>
  </footer>
</article>
`;
